js/_website/CHANGELOG.md

```diff
@@ -38,13 +38,18 @@
 
 ### Dependency updates
 
+<<<<<<< HEAD
```

Comment:
Looks like there are a few conflicts in the changelogs that didn't get resolved?

Reply:
This is so weird, it happened a few times. Will check them.